pkg/rpc/server/listen.go

```diff
@@ -121,7 +121,11 @@ func newDBWorker(dbClient dbFile.Operation) dbWorker {
 }
 
 func (w dbWorker) update(ctx context.Context, appVersion, cacheDir string,
-	dbUpdateWg, requestWg *sync.WaitGroup) error {
+	skipDbUpdate bool, dbUpdateWg, requestWg *sync.WaitGroup) error {
```

Comment:
nit: Db -> DB
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

```suggestion
	skipDBUpdate bool, dbUpdateWg, requestWg *sync.WaitGroup) error {
```

Reply:
Done